modules/ROOT/pages/flex-gateway-monitoring.adoc

```diff
@@ -0,0 +1,23 @@
+= Monitoring
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]s
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+You can see the details of the logs of your Flex Gateway in Anypoint Monitoring.
```

Comment:
suggestion to avoid using modal verb "can": 

Anypoint Monitoring shows the details of the logs of your Flex Gateway.

Reply:
Applied!